standards-in-genomic-sciences.csl

```diff
@@ -182,7 +182,7 @@
           <text macro="editor" prefix=" " suffix="."/>
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short" suffix=" "/>
-            <group delimiter=";" prefix=" ">
+            <group delimiter=";">
               <date variable="issued">
                 <date-part name="year"/>
               </date>
```

Comment:
another thing we could consider removing via script: groups with a single element. Sed probably isn't a good choice here, but python should work nicely. cc @rmzelle